JASP-Desktop/po/jasp_nl.po

```diff
@@ -16078,10 +15845,2124 @@ msgctxt "RibbonModel|"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "Transform estimates"
+msgstr "Transformeer schattingen"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "Advanced"
+msgstr "Geavanceerd"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "Transform correlations"
+msgstr "Getransformeerde correlatie"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "Cohen's d"
+msgstr "Cohen's d"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "Fisher's z"
+msgstr "Fisher's z"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "Bridge sampling iterations:"
+msgstr "Bridge sampling iteraties:"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "Estimation settings (MCMC)"
+msgstr "Schattingsinstellingen (MCMC)"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "Adaptation"
+msgstr "Aanpassing"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "Burnin"
+msgstr "Opwarming"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "Iterations"
+msgstr "Iteraties"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "Chains"
+msgstr "Ketens"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "Thin"
+msgstr "Dun"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "Autofit"
+msgstr "Autopassing"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "Maximum fitting time"
+msgstr "Maximale pastijd"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "hours"
+msgstr "uren"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "minutes"
+msgstr "minuten"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "seconds"
+msgstr "seconden"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "Target margin of error"
+msgstr "Doelwit foutmarge"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "Exclude models"
+msgstr "Sluit modellen uit"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "error % >"
+msgstr "fout % >"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "R-hat >"
+msgstr "R-hat >"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "Estimated sample size <"
+msgstr "Geschatte steekproefgrootte <"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "Redistribute prior probability"
+msgstr "Herverdeel prior kans"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "Conditional models"
+msgstr "Conditioneel modellen"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "Model space"
+msgstr "Modelruimte"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "Control"
+msgstr "Controle"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "Clever refitting"
+msgstr "Slim opnieuw passen"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "Do not refit"
+msgstr "Pas het niet opnieuw"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "Always refit"
+msgstr "Pas het altijd opnieuw"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "Save the fitted model"
+msgstr "Sla de gefitte model op"
+
+msgctxt "MixedModelsBGLMM|"
+msgid "Dependent variable"
+msgstr "Afhankelijke variabele"
+
+msgctxt "MixedModelsBGLMM|"
+msgid "Number of trials"
+msgstr "Aantal proeven"
+
+msgctxt "MixedModelsBGLMM|"
+msgid "Fixed effects variables"
+msgstr "Vaste-effecten variabelen"
```

Comment:
Op andere plaatsen gebruiken we "gefixeerde" ipv "vast" (e.g., "Gefixeerde effecten"). Dunno wat beter is

Reply:
Ik merk dat in ANCOVA/ANOVA het vaste effecten zijn.

Reply:
Alles gefixeerde effecten